### ISSUE TITLE
false positive when checking fsf_wrong_address

### ISSUE THREAD

**Issue by subpop** [rpm-software-management/rpmlint#85]
I ran rpmlint on a package and it printed the `incorrect-fsf-address` warning on the [COPYING](https://github.com/freeorion/freeorion/blob/master/default/COPYING) file. I looked into the file, and it appears to have the current address.

When I run the regex against the file manually using `grep` and it does not return a match:

```
 grep -E '(675\s+Mass\s+Ave|59\s+Temple\s+Place|Franklin\s+Steet|02139|02111-1307)' src-tarball/default/COPYING
```

Is this a false positive?

**Comment by scop** [comment]
Hard to tell without access to the actual package being checked. Is it available somewhere online?

**Comment by subpop** [comment]
Yea. It's in the Fedora Updates testing repo. Here is a link to the build page. You should be able to download the x86_64.rpm and the src.rpm. http://koji.fedoraproject.org/koji/buildinfo?buildID=821136

**Comment by scop** [comment]
The COPYING file you linked to is not the one included in the package; the one in it is an old version of the LGPL 2.1 which does contain an old FSF address.

```shell
$ curl -O https://kojipkgs.fedoraproject.org//packages/freeorion/0.4.6/4.fc25/x86_64/freeorion-0.4.6-4.fc25.x86_64.rpm
$ rpmdev-extract freeorion-0.4.6-4.fc25.x86_64.rpm
$ grep -E '(675\s+Mass\s+Ave|59\s+Temple\s+Place|Franklin\s+Steet|02139|02111-1307)' freeorion-0.4.6-4.fc25.x86_64/usr/share/licenses/freeorion/COPYING
     59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
```

**Comment by subpop** [comment]
I found that. There are two `COPYING` files that I was including as the `%license` in the spec file. One of them does have the old address. Thanks for pointing that out!
